MapboxMobileEvents/MMEMetrics.m

```diff
@@ -5,24 +5,29 @@ @implementation MMEMetrics
 - (instancetype)init {
     self = [super init];
     if (self) {
-        self.failedRequestsDict = [[NSMutableDictionary alloc] init];
-        self.eventCountPerType = [[NSMutableDictionary alloc] init];
-        self.date = [NSDate date];
-        self.totalDataTransfer = 0;
-        self.cellDataTransfer = 0;
-        self.wifiDataTransfer = 0;
-        self.totalDataReceived = 0;
-        self.cellDataReceived = 0;
-        self.wifiDataReceived = 0;
-        self.eventCountFailed = 0;
-        self.eventCountTotal = 0;
-        self.eventCountMax = 0;
-        self.appWakeups = 0;
-        self.deviceLat = 0;
-        self.deviceLon = 0;
-        self.requests = 0;
+        _failedRequestsDict = [[NSMutableDictionary alloc] init];
+        _eventCountPerType = [[NSMutableDictionary alloc] init];
+        _date = [NSDate date];
+        _totalBytesSent = 0;
+        _cellBytesSent = 0;
+        _wifiBytesSent = 0;
+        _totalBytesReceived = 0;
+        _cellBytesReceived = 0;
+        _wifiBytesReceived = 0;
+        _eventCountFailed = 0;
+        _eventCountTotal = 0;
+        _eventCountMax = 0;
+        _appWakeups = 0;
+        _deviceLat = 0;
+        _deviceLon = 0;
+        _requests = 0;
     }
     return self;
 }
 
+- (void)computeTransferredBytes {
```

Comment:
Suggest removing the `_total…` properties and adding:

`- (NSUInteger) totalBytesSent { return _wifiBytesSent + _cellBytesSent; }`

This will satisfy the requirements for the `@property` directive and you won't have to remember to call this compute method, or store the product until it's needed for an event report.